Code-the-Dream-School/cc-prac-team4-front
# PP-92 (Part #1): Create select options for pets page

I created, connected with back-end and added functionality to couple select options elements on pets page:

1. Pet type: cat/dog
2. Pet breed:  list of cat's breeds/list of dog's breeds

Second one(breeds) depending on the user selection in first elements(type).
The list for cat breeds and dog breeds are obtained from two different API requests.   

**Modified files:**

- src/actions/petAction.js
- src/component/Pet/Pets.js
- src/component/Pet/Pets.module.css
- src/reducers/petReducer.js

**Steps for testing:**
1. Open pets page [http://localhost:3000/pets](http://localhost:3000/pets)
2. Make sure the Type field is able and field Breed disabled

<img width="214" alt="Screenshot 2023-06-16 at 9 27 44 PM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/2952900/7775a13e-a5a8-4734-8c78-4ed7c8d5eb4e">

3. Make sure the Breed field becomes available as soon as the Type field(Cat or Dog) is selected.
4. Make sure that when cats are selected in the Type field, the breeds of cats are displayed in the Breeds field, and vice versa, when dogs are selected in the Type field, dog breeds are displayed in the Breed field.

<img width="259" alt="Screenshot 2023-06-16 at 9 27 30 PM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/2952900/0235657f-67e4-420e-a810-bf7e12481d6b">
<img width="329" alt="Screenshot 2023-06-16 at 9 27 57 PM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/2952900/de909204-401f-4ce1-8c18-0783e65edd75">
